src/ms/validation.php

```diff
@@ -92,7 +92,7 @@
     'not_in'               => ':attribute tidak sah.',
     'not_regex'            => 'Format :attribute adalah tidak sah.',
     'numeric'              => ':attribute mesti nombor.',
-    'password'             => 'The password is incorrect.',
+    'password'             => ':attribute tidak sah.',
```

Comment:
Why do you add :attribute ?

Reply:
so no need attribute? let me change to 'Kata laluan tidak sah' then

Reply:
The original text of this line indicates that the password is incorrect. It has no reference to the attribute name.

Reply:
@andrey-helldar ok noted. already fixed it